git_machete/tests/functional/test_machete.py

```diff
@@ -2799,3 +2799,40 @@ def test_add(self) -> None:
         self.assert_command(['add', '--onto=feature'],
                             'Added branch `chore/remove_indentation` onto `feature`\n',
                             strip_indentation=False)
+
+    @mock.patch('git_machete.utils.run_cmd', mock_run_cmd)
+    def test_clean(self) -> None:
+        (
+            self.repo_sandbox.new_branch('master')
+                .commit()
+                .push()
+                .new_branch('o-bar')
```

Comment:
Oh! `o-` served as an green-edge _elbow_ in my example... see `git machete status --color=never`! + its docs
It wasn't meant as a part of branch name